apps/admin/backend/src/app.ts

```diff
@@ -254,12 +254,11 @@ function buildApi({
 
     /* istanbul ignore next */
     async generateSignedHashValidationQrCodeValue() {
-      const { codeVersion, machineId } = getMachineConfig();
+      const { codeVersion } = getMachineConfig();
```

Comment:
`getMachineConfig` does still read machine ID from the `VX_MACHINE_ID` env var which reads from the `/vx/config/machine-id` file. We could eventually get rid of the `/vx/config/machine-id` file and always pull the machine ID from the machine cert. But for the few touch points where it truly matters, we are reading from the cert (after verifying the cert) (as you'll see below).

It ballooned scope too much under our time constraints to completely get rid of the `/vx/config/machine-id` file. But I will ticket that future change and same for the analogous change for jurisdiction ID on VxAdmin.

Reply:
as long as we're verifying the machine ID in certs at signature verification time, I'm not worried about this. Yes, for simplicity and consistency, good to fix this later, but not urgent in this PR.

Reply:
Can we mark the property as deprecated, and link to the issue you (will have) created?

```ts
export interface MachineConfig {
  /**
   * @deprecated Use the machine ID from the machine's certificate instead.
   */
  machineId: string;
```

Reply:
Looking at this again, I'm not necessarily thinking of this field as deprecated. It's okay for us to group machine ID with code version in this interface, and to have people access machine ID through that interface. We just want to be populating the field differently, extracting from the machine cert instead of relying on the VX_MACHINE_ID env var, which is currently populated by the `machine-id` file in vxsuite-complete-system:

https://github.com/votingworks/vxsuite-complete-system/blob/78f0858f525def8bfc0c08d84a77a4808649cc7b/config/read-vx-machine-config.sh#L43

That said, we may still arrive at the conclusion that we want to remove the field from this interface.

I think it's a bit too early for me to tell without considering how I want to approach this holistically, across machines, machine ID + jurisdiction, and vxsuite + vxsuite-complete-system.

So I'm not going to add the deprecated marker yet.

I've grouped this together with the existing ticket I had for jurisdiction and updated accordingly: https://github.com/votingworks/vxsuite/issues/3322